regolith/helpers/a_proposalhelper.py

```diff
@@ -157,9 +169,9 @@ def db_updater(self):
             pdoc.update({'pi': rc.pi})
         else:
             pdoc.update({'pi': rc.pi_id})
-        pdoc.update({'status': 'inprep'})
+        pdoc.update({'status': 'in-prep'})
```

Comment:
I would rather have inprep.  If this means a schema change, let's do the shema change.  It is always a headache to keep track of uppercase letters and dashes and underscores in allowed values, so my go-to rule is not use them unless really necessary.

Reply:
Ok! I will add `inprep` to `ALLOWED_STATI` but leave `in-prep` in for backwards compatibility

Reply:
Thanks!  Please remove ``in-prep``  Your decision was a good one but in-prep is mine so there are no backwards compatibility issues further back than when I took over....

Reply:
done!